buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -90,4 +94,31 @@ class PrecommitTasks {
             rule name: 'tab', pattern: /\t/
         }
     }
+
+    static Task configureJarHell(Project project) {
+        File successMarker = new File(project.buildDir, 'markers/jarHell')
```

Comment:
what is this successMarker and why is it only "set" if logger is not at info level.... that is a little confusing?